mix.exs

```diff
@@ -80,7 +80,9 @@ defmodule FunWithFlags.Mixfile do
 
       {:ex_doc, "~> 0.21", only: :dev, runtime: false},
       {:credo, "~> 1.7", only: :dev, runtime: false},
-      {:dialyxir, "~> 1.0", only: :dev, runtime: false}
+      {:dialyxir, "~> 1.0", only: :dev, runtime: false},
+      {:telemetry_test, "~> 0.1", only: :test, runtime: false},
```

Comment:
I'm not sure about this package. It looks like it's not really much used in the ecosystem: https://hex.pm/packages/telemetry_test

Is there any other way to test telemetry?

Reply:
yes for sure,[]( https://hex.pm/packages/telemetry_test ) just abstract away the setup for telemetry event handler. But now i have updated the tests without using the telemetry_test package.